.gitignore

```diff
@@ -1,343 +1 @@
-DocProject/Help/Html2
-DocProject/Help/html
-
-# Click-Once directory
-publish/
-
-# Publish Web Output
-*.[Pp]ublish.xml
-*.azurePubxml
-
-# TODO: Un-comment the next line if you do not want to checkin
-# your web deploy settings because they may include unencrypted
-# passwords
-#*.pubxml
-*.publishproj
-
-# NuGet Packages
-*.nupkg
-# The packages folder can be ignored because of Package Restore
-**/packages/*
-# except build/, which is used as an MSBuild target.
-!**/packages/build/
-# Uncomment if necessary however generally it will be regenerated when needed
-#!**/packages/repositories.config
-# NuGet v3's project.json files produces more ignoreable files
-*.nuget.props
-*.nuget.targets
-
-# Microsoft Azure Build Output
-csx/
-*.build.csdef
-
-# Microsoft Azure Emulator
-ecf/
-rcf/
-
-# Microsoft Azure ApplicationInsights config file
-ApplicationInsights.config
-
-# Windows Store app package directory
-AppPackages/
-BundleArtifacts/
-
-# Visual Studio cache files
-# files ending in .cache can be ignored
-*.[Cc]ache
-# but keep track of directories ending in .cache
-!*.[Cc]ache/
-
-# Others
-ClientBin/
-[Ss]tyle[Cc]op.*
-~$*
-*~
-*.dbmdl
-*.dbproj.schemaview
-*.pfx
-*.publishsettings
-node_modules/
-orleans.codegen.cs
-
-# RIA/Silverlight projects
-Generated_Code/
-
-# Backup & report files from converting an old project file
-# to a newer Visual Studio version. Backup files are not needed,
-# because we have git ;-)
-_UpgradeReport_Files/
-Backup*/
-UpgradeLog*.XML
-UpgradeLog*.htm
-
-# SQL Server files
-*.mdf
-*.ldf
-
-# Business Intelligence projects
-*.rdl.data
-*.bim.layout
-*.bim_*.settings
-
-# Microsoft Fakes
-FakesAssemblies/
-
-# GhostDoc plugin setting file
-*.GhostDoc.xml
-
-# Target VS files:
-vsx64
-
-# Node.js Tools for Visual Studio
-.ntvs_analysis.dat
-
-# Visual Studio 6 build log
-*.plg
-
-# Visual Studio 6 workspace options file
-*.opt
-
-# Visual Studio LightSwitch build output
-**/*.HTMLClient/GeneratedArtifacts
-**/*.DesktopClient/GeneratedArtifacts
-**/*.DesktopClient/ModelManifest.xml
-**/*.Server/GeneratedArtifacts
-**/*.Server/ModelManifest.xml
-_Pvt_Extensions
-
-# LightSwitch generated files
-GeneratedArtifacts/
-ModelManifest.xml
-
-# Paket dependency manager
-.paket/paket.exe
-
-# FAKE - F# Make
-.fake/
-*.filters
-/External
-/Output
-/InferenceEngineMain/models
-/Test
-/HTTPClient/*.a
-/InferenceEngineMain/newModels
-.DS_Store
-
-# For IDEA
-.idea/
-VS/
-Xcode/
-temp/
-report/
-.kdev4/
-*.kdev4
-*.kate-swp
-
-/lin-build
-/win-build
-/CMakeFiles
-*.stamp
-*.depend
-*.vcxproj
-*.sln
-/CMakeCache.txt
-.vimprj/
-build_IA32/
-doc/
-.dir-locals.el
-GTAGS
-GPATH
-GRTAGS
-GSYMS
-compile_commands.json
-service/dot-net-service/Output
-**/sublime_build
-/.project
-.vscode/
-/vsx32
-/service/dot-net-service/.klocwork/DotNetService
-cmake-build-*/
-/lin64
-
-.gdb_history
-.local_vimrc
-.ycm_extra_conf.py
-tags
-
-
-# from Model Optimizer repo
-.idea
-.project
-.cproject
-.pydevproject
-.settings
-/bin/
-/gen/
-__pycache__
```

Comment:
It makes sense to keep `__pycache__` for Python 3

Reply:
`*.pyc` is sufficient. `__pycache__` directories contain only `.pyc` files, and Git doesn't notice directories that only contain ignored files.